common/app/model/content.scala

```diff
@@ -429,6 +429,8 @@ private object ArticleSchemas {
     // http://schema.org/Review
     if (article.isReview)
       "http://schema.org/Review"
+    else if (article.isLive)
+      "http://schema.org/LiveBlogPosting"
```

Comment:
I did mention it doesn't validate due to the schema.org not being up yet :laughing: